Kalliope-Import-Plugin

solth opened the pull request:
Import metadata for documents via the Kalliope SRU interface

stweil commented on Goobi/plugins/opac/ModsPlugin/org/kitodo/production/plugin/CataloguePlugin/ModsPlugin/UGHUtils.java:
```diff
@@ -0,0 +1,249 @@
+/*
```
Would it be better to add this file to Kitodo.ugh?

henning-gerhardt replied:
As this file contains plugin specific code, this file should be stay inside plugin. There is a identical named file in PicaPlugin too.

stweil commented on Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/PicaPlugin.java:
```diff
@@ -393,10 +411,11 @@ public static String getDescription(Locale language) {
 		return createResult(cod.getTitle(), myFirstHit, ff);
 	}
-
+	
```
Please remove any spaces at line endings.

henning-gerhardt commented on Goobi/src/org/goobi/production/plugin/CataloguePlugin/CataloguePlugin.java:
```diff
@@ -131,6 +139,11 @@
 	/**
+	 * Local log4j logger.
+	 */
+	private static final Logger logger = Logger.getLogger(ConfigOpac.class);
```
Class name is `CataloguePlugin` and logger should be identical.

henning-gerhardt commented on Goobi/src/de/sub/goobi/forms/ProzesskopieForm.java:
```diff
@@ -1573,6 +1669,14 @@ public void setOpacSuchbegriff(String opacSuchbegriff) {
 		this.opacSuchbegriff = opacSuchbegriff;
 	}
 
+	public String getInstitution() {
```
A short JavaDoc entry should be nice for this getter Method.

henning-gerhardt commented on Goobi/src/de/sub/goobi/forms/ProzesskopieForm.java:
```diff
@@ -1573,6 +1669,14 @@ public void setOpacSuchbegriff(String opacSuchbegriff) {
+		return institution;
+	}
+
+	public void setInstitution(String institution) {
```
A short JavaDoc entry should be nice for this stter Method.

henning-gerhardt commented on Goobi/plugins/opac/ModsPlugin/org/kitodo/production/plugin/CataloguePlugin/ModsPlugin/package-info.java:
```diff
@@ -0,0 +1,37 @@
+ *
+ * @author Arved Solth, Christopher Timm
+ */
+package org.kitodo.production.plugin.CataloguePlugin.ModsPlugin;
```
Line break at end of file would be nice.

henning-gerhardt commented on Goobi/plugins/opac/ModsPlugin/org/kitodo/production/plugin/CataloguePlugin/ModsPlugin/kalliope2kitodo.xsl:
```diff
@@ -0,0 +1,116 @@
+    <xsl:template match="@*|node()">
+        <xsl:apply-templates select="@*|node()"/>
+    </xsl:template>
+</xsl:stylesheet>
```
Line break at end of file would be nice.

stweil commented on Goobi/config/kitodo_mods_opac.xml:
```diff
@@ -0,0 +1,57 @@
+			<detail name="url" xpath="//goobi:metadata[@name='slub_link']" />
+		</additionalDetails>
+	</catalogue>
+</opacCatalogues>
```
Missing LF in last line.

stweil commented on Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/PicaPlugin.java:
```diff
@@ -45,6 +49,8 @@
 import org.joda.time.LocalTime;
 import org.w3c.dom.Node;
 
+import net.xeoh.plugins.base.Plugin;
```
Sort import statements alphabetically?

solth commented:
- added javadoc to functions "getInstitution" and "setInstitution" in class "Prozesskopieform"
- removed trailing whitespace from PicaPlugin.java
- added LF to files kalliope2kitodo.xsl, package-info.java and kitodo_mods_opac.xml 
- updated logger name in CataloguePlugin.java

henning-gerhardt commented:
@stweil: is something left over which must be corrected or is PR ready to merge?
